pkg/skaffold/util/util_test.go

```diff
@@ -468,3 +469,31 @@ func TestIsURL(t *testing.T) {
 func stringPointer(s string) *string {
 	return &s
 }
+
+func TestShowHumanizeTime(t *testing.T) {
+	currTime := time.Now()
+	duration1, _ := time.ParseDuration("1h58m30.918273645s")
+	duration2, _ := time.ParseDuration("5.23494327s")
+	tests := []struct {
+		description string
+		value       time.Time
+		expected    string
+	}{
+		{
+			description: "Case for 1h58m30.918273645s",
+			value:       currTime.Add(-duration1),
+			expected:    "1 hour 58 minutes 30.9",
+		},
+		{
+			description: "Case for 5.23494327s",
+			value:       currTime.Add(-duration2),
+			expected:    "5.2",
+		},
+	}
+	for _, test := range tests {
+		testutil.Run(t, test.description, func(t *testutil.T) {
+			humanizedValue := ShowHumanizeTime(test.value)
+			t.CheckMatches(test.expected, humanizedValue)
```

Comment:
In this case even if ```ShowHumanizeTime``` returned "5.23412341" (which is not "humanized") the test would still pass since ```t.CheckMatches``` does ```regexp.MatchString("5.2","5.23412341")```.

You should either make your pattern more strict or use ```t.CheckDeepEquals``` instead